<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AskDlg</class>
 <widget class="QDialog" name="AskDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>131</width>
    <height>101</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>131</width>
    <height>101</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>131</width>
    <height>101</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblNum">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>label</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbVal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vul in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuleer</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>130</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="leNum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>lblNum</zorder>
  <zorder>pbVal</zorder>
  <zorder>pbCancel</zorder>
 </widget>
 <tabstops>
  <tabstop>leNum</tabstop>
  <tabstop>pbVal</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
